src/helpers/keyboardHelper.ts

```diff
@@ -0,0 +1,25 @@
+import { useEffect, useState } from 'react';
+import { Keyboard, KeyboardEvent } from 'react-native';
+
+export const getKeyboardHeight = () => {
```

Comment:
this seems to be a hook and should be placed in hooks and should be named `useKey...`

Reply:
Yes, you are right. I've updated.

https://github.com/smart-village-solutions/smart-village-app-app/commit/e1405deffc70311423aee4034dfc441aee5c4eba